### DISCUSSION TITLE
Using Safari results in an immediate login failure when trying to sign in with Google

### DISCUSSION THREAD

**Discussion by sagelga** [monkeytypegame/monkeytype#5530] [Ideas] [answered]
When attempting to sign in to MonkeyType using Google on Safari, the sign-in process is disrupted by Safari’s default settings, which automatically disable any popup windows. This prevents me from completing the sign-in process. 

The current workaround involves manually enabling pop-up windows for the MonkeyType website. However, it would be more user-friendly to have a sign-in process that does not require manual intervention to enable pop-ups.

So I am suggesting that MonkeyType should implement a sign-in process that bypasses the need for pop-up windows.

**Comment by Miodec** [comment]
Unfortunately Firebase Auth doesn't have a way (as far as i know) to authenticate with google without a popup.

**Reply by Whydidthechickencrosstheroad** [reply, marked as answer]
Try making it open a new tab @Miodec instead

**Reply by sagelga** [reply]
> Try making it open a new tab @Miodec instead

Seems like a reasonable workaround! But I am not so sure how other big companies differ from the Sign-in with Google and can get away with this Safari limitation.

**Reply by sagelga** [reply]
@Miodec please take this question as a feature request. This little change will going to speed up login process for all Safari users.

**Reply by Miodec** [reply]
Ill update the error message to mention the user has to allow the popup.
